### PR TITLE
MEDIUM FIX: Support `application-json`

### PR DESCRIPTION
**Pull request by BrianLParker**
#50

### CODE REVIEW

**Comment by BrianLParker** [comment]
Guys this is a version bump on **BOTH** Packages

File: RESTFulSense.WebAssembly/Clients/RESTFulApiClient.cs [hunk 1]
```diff
@@ -65,20 +63,26 @@ public async ValueTask<T> GetContentAsync<T>(string relativeUrl, CancellationTok
         public ValueTask<T> PostContentAsync<T>(
             string relativeUrl,
             T content,
-            string mediaType = "text/json") => PostContentAsync<T, T>(relativeUrl, content, mediaType);
+            string mediaType = "text/json")
+        {
```
Comment (cjdutoit) [inline RESTFulSense.WebAssembly/Clients/RESTFulApiClient.cs:67]: We can change this to use `=>` as its a one liner.

File: RESTFulSense.WebAssembly/Clients/RESTFulApiClient.cs [hunk 2]
```diff
@@ -65,20 +63,26 @@ public async ValueTask<T> GetContentAsync<T>(string relativeUrl, CancellationTok
...
+        }
 
         public ValueTask<T> PostContentAsync<T>(
             string relativeUrl,
             T content,
             CancellationToken cancellationToken,
-            string mediaType = "text/json") => PostContentAsync<T, T>(relativeUrl, content, cancellationToken, mediaType);
+            string mediaType = "text/json")
+        {
+            return PostContentAsync<T, T>(relativeUrl, content, cancellationToken, mediaType);
```
Comment (cjdutoit) [inline RESTFulSense.WebAssembly/Clients/RESTFulApiClient.cs:77]: We can change this to use `=>` as its a one liner.

**Review by cjdutoit — CHANGES_REQUESTED** [review]
Some minor changes needed

**Pull request merged**